### DISCUSSION TITLE
807

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#982] [Comments]
# 807

주선님 과 유진님 이 운영하는 독서모임의 두번째 책이 The Goal (더 골)(만화판) 이다.미스터 비스트가 추천하는 책으로 유명하지만, IT 개발자분들에게는 오히려 피닉스 프로젝트로 더 유명할 것 같다.그래서 양 책을 함께 읽으면서 제조업에서의, IT 업에서의 TOC 를 비교해가며 재밌게 읽을 수 있다.피닉스 프로젝트를 보고 나서 너무 책이 재밌어서 더 골 역시 재밌게 읽었다.이번 모임 책으로 선정되서 다시 읽어보니 과거와는 다르게 읽혀졌다.실무자의 입장에서 읽었을 때와 리더 포지션으로 읽을때랑은 많은 부분이 다르게 느껴졌다.아래는 책을 보며 인상 깊었던 구절들을 옮겼다.회사의 목표인건비도, 재고도 줄지 않고, 제품의 출하량도 늘어나지 않았다면 '생산성이 향상됐다' 고는 말할 수 없다.생산성이란 회사..

https://jojoldu.tistory.com/807?utm_campaign=asb&utm_medium=blog&utm_source=awesome-blogs.petabytes.org

**Comment by nanamix** [comment]
헉!!! 책으로만 읽었던 The Goal 이.. 코믹스버전이 있다니.. (사실 텍스트는 참 힘들긴... 했죠. ㅎㅎㅎ)

다시 한번 사서 일어 봐야 겠네요!!!
